ui/app/mirrors/create/cdc/columnbox.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+import { TableMapRow } from '@/app/dto/MirrorsDTO';
+import { Checkbox } from '@/lib/Checkbox';
+import { Label } from '@/lib/Label';
+import { RowWithCheckbox } from '@/lib/Layout';
+import { Dispatch, SetStateAction } from 'react';
+
+interface ColumnProps {
+  columns: string[];
+  tableRow: TableMapRow;
+  rows: TableMapRow[];
+  setRows: Dispatch<SetStateAction<TableMapRow[]>>;
+}
+const ColumnBox = ({ columns, tableRow, rows, setRows }: ColumnProps) => {
```

Comment:
```suggestion
export default function ColumnBox({ columns, tableRow, rows, setRows }: ColumnProps) {
```
Not sure what motivation is to declare as lambda & then `export default` it